app/models/user.rb

```diff
@@ -0,0 +1,3 @@
+class User < ActiveRecord::Base
+  validates :username, :token, presence: true
```

Comment:
The presence of email cannot be validate for now, because Github is not sending this field, is just created in case that in the future we would need it